src/main/cpp/t02_up.cpp

```diff
@@ -24,5 +24,12 @@
 using namespace std;
 
 int t02_up() {
-    
+    char s1,s2;
+    int diff=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'diff' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4487945)